docbook/Developers_Guide/en-US/Events_Reference_Bug.xml

```diff
@@ -277,6 +317,126 @@
 			</blockquote>
 		</blockquote>
 
+		<blockquote id="dev.eventref.bug.action.reportbugpre">
+			<title>EVENT_BUG_CREATE_PRE (Execute)</title>
+
+			<blockquote>
+				<para>
+					This event allows plugins to perform pre-processing of a new bug data
+					structure before the data is saved to the database. At this point,
+					the issue ID is not yet known, as the data has not yet been persisted.
+					Modifications on this structure can be made in the BugData object
+					received as parameter.
+					This event is triggered by the BugData API by any action that uses
+					the create() method.
+				</para>
+
+				<para>
+					Note: As the create action may be altered or stopped by other plugins,
```

Comment:
How should plugins trigger the stopping in a way that works well with web UI and APIs?  Is it via trigger_error?  It is worth testing this case and clarifying guidance in docs as needed.